server/controller/modules.js

```diff
@@ -15,10 +15,18 @@ class ModulesController {
    */
   static async getAllModules(req, res) {
     try {
-      const modules = await Module.findAll();
-      return res.status(200).json({
-        status: 200,
-        data: modules
+      const { userId } = req.user;
+      const user = await User.findByPk(userId);
```

Comment:
This doesn't seem right, I'm not sure why you need to fetch the `user` so as to return the `modules`

Could explain more, please?

Reply:
Looking at it again, I agree with you. It doesn't seem right since there is already a middleware checking for token.